### DISCUSSION TITLE
Is it possible to link observable state to observer at runtime?

### DISCUSSION THREAD

**Discussion by kyunghoon-paik** [mobxjs/mobx#2660] [Q&A]
Hello everyone!

Is it possible to attach observable state to a specific observer at runtime?

My situation is, among observers created with the same code, only a "specific" observer, which is determined at runtime, need to be attached with observable state.

plz help me!

**Comment by mweststrate** [comment]
Could you elaborate with a code example?

**Comment by urugator** [comment]
https://github.com/mobxjs/mobx-react/issues/667#issuecomment-483747386
